src/waveform/renderers/waveformsignalcolors.cpp

```diff
@@ -134,6 +140,30 @@ void WaveformSignalColors::fallBackDefaultColor() {
     fallBackFromSignalColor();
 }
 
+QColor WaveformSignalColors::rgbaColorFromString(QString sColorString) {
```

Comment:
I think it would be straight forward to move this function to skincontext.h similar to 
hasNodeSelectString() 

something like 

hasNodeSelectColor() 

This way all color strings in the skin can benefit from the #AARRGGBB syntax.